packages/grid/x-data-grid/src/tests/cells.DataGrid.test.tsx

```diff
@@ -184,6 +185,37 @@ describe('<DataGrid /> - Cells', () => {
     }).toWarnDev(['MUI: The cell with id=1 and field=brand received focus.']);
   });
 
+  it('Should be in the DOM even if when the focused cell is out of viewport.', function test() {
```

Comment:
Start with a lowercase, like the other test labels, no final period. The sentence could also be reworked to be a bit more clear.